docs/user-guide/src/capm3/installation_guide.md

```diff
@@ -11,7 +11,7 @@ install them yourself.
 
 1. Install `clusterctl`, refer to Cluster API [book](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl) for installation instructions.
 1. Install `kustomize`, refer to official instructions [here](https://kubectl.docs.kubernetes.io/installation/kustomize/).
-1. Install Ironic, refer to [this page](../ironic/ironic_installation.html).
+1. Install Ironic, refer to [this page](https://github.com/metal3-io/metal3-docs/blob/main/docs/user-guide/src/ironic/ironic_installation.md).
```

Comment:
This link is not broken, why change it? It refers to Ironic installation in the user guide book.

Reply:
I changed it because when i clicked on the link, it returned a 404 page not found. here's a screenshot of the link destination:
![404](https://github.com/metal3-io/metal3-docs/assets/111025771/d3f726bb-2439-43f2-8299-26f3de211e84)

Reply:
It is not relative to the repository, but to the user-guide generated site. Use the generated preview site to check if links work or not. The broken links are listed in the issue: https://github.com/metal3-io/metal3-docs/issues/328

Reply:
thanks for this, it'll help me so much going forward, i'll take it out